token/cli/src/main.rs

```diff
@@ -5308,6 +5603,125 @@ async fn process_command<'a>(
             )
             .await
         }
+        (CommandName::ConfigureConfidentialTransferAccount, arg_matches) => {
+            let (owner_signer, owner) =
+                config.signer_or_default(arg_matches, "owner", &mut wallet_manager);
+
+            let token_account =
+                config.pubkey_or_default(arg_matches, "account", &mut wallet_manager)?;
+
+            // Deriving ElGamal and AES key from signer. Custom ElGamal and AES keys will be
+            // supported in the future once upgrading to clap-v3.
+            let elgamal_keypair =
+                ElGamalKeypair::new_from_signer(&*owner_signer, &token_account.to_bytes()).unwrap();
+            let aes_key =
+                AeKey::new_from_signer(&*owner_signer, &token_account.to_bytes()).unwrap();
+
+            if config.multisigner_pubkeys.is_empty() {
+                push_signer_with_dedup(owner_signer, &mut bulk_signers);
+            }
+
+            let maximum_credit_counter = value_t!(
+                arg_matches.value_of("maximum_pending_balance_credit_counter"),
+                u64
+            )
+            .ok();
```

Comment:
Unless I'm missing something, if someone puts in an invalid number here, it'll silently fallback to the default. You'll probably want to properly handle the error if the arg is present, ie:

```
arg_matches.value_of("maximum_pending_balance_credit_counter").map(|v| v.parse::<u64>().unwrap_or_else(print_error_and_exit))

Reply:
Yes, this is a good point. In clap-v3, this will panic, but in clap-v2, this will set `maximum_credit_counter` to default. Fixed.